homework2/main.py

```diff
@@ -0,0 +1,265 @@
+import requests
+from collections import Counter
+from datetime import datetime, timedelta
+import csv
+import copy
+
+
+class MovieDataGrabber:
+    def __init__(self, api_key, num_pages=1):
+        self.api_key = api_key
+        self.num_pages = num_pages
+        self.url = "https://api.themoviedb.org/3/discover/movie"
+        self.data = None
```

Comment:
I would suggest you to set empty list right here and then apply extend  directly on this field